src/modules/emote_menu/components/Button.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, {useEffect, useRef} from 'react';
+import Whisper from 'rsuite/lib/Whisper/index.js';
+import Popover from 'rsuite/lib/Popover/index.js';
+import EmoteMenu from './Menu.jsx';
+import styles from '../styles/button.module.css';
+import emoteStore from '../stores/index.js';
+
+export default function Button({appendToChat, setPopoverOpen}) {
+  const triggerRef = useRef(null);
+
+  useEffect(() => {
+    if (emoteStore.isLoaded()) {
+      setPopoverOpen(triggerRef);
+    }
+
+    emoteStore.once('loaded', () => {
+      setPopoverOpen(triggerRef);
+    });
+  }, []);
+
+  return (
+    <Whisper
+      trigger="active"
+      placement="topEnd"
+      speaker={
+        <Popover className={styles.popover} full>
+          <EmoteMenu triggerRef={triggerRef} appendToChat={appendToChat} />
```

Comment:
![image](https://user-images.githubusercontent.com/2942184/131237030-dad195e6-f11a-4be8-bf1b-5304de5d0a63.png)

the popout arrow does not align to the button